data_safe_haven/administration/users/active_directory_users.py

```diff
@@ -110,3 +110,14 @@ def register(self, sre_name: str, usernames: Sequence[str]) -> None:
         )
         for line in output.split("\n"):
             self.parse_as_log(line)
+
+    def remove(self, users: Sequence[ResearchUser]) -> None:
+        """Remove list of users from local Active Directory"""
+        pass
```

Comment:
```suggestion
```

Reply:
Sorry - I have implementations for these stubs but they aren't quite working atm.